### ISSUE TITLE
Translations with state "new" in the xliff gets imported

### ISSUE THREAD

**Issue by jeeyi** [box/mojito#390]
@aurambaj Do you remember why we chose to import translations with state "new" when importing xliffs? If we import what was just exported without making changes, English source strings get saved as translations. Is this intended behavior?

**Comment by aurambaj** [comment]
Are you talking about `ImportExportedXliffStep`, isn't that supposed to read the status from the XLIFF and keep whatever status was in the repository at the time of export?

**Comment by jeeyi** [comment]
I am talking about `AbstractImportTranslationsStep`: https://github.com/box/mojito/blob/master/webapp/src/main/java/com/box/l10n/mojito/okapi/AbstractImportTranslationsStep.java#L297-L319

I am thinking about changing https://github.com/box/mojito/blob/master/webapp/src/main/java/com/box/l10n/mojito/okapi/AbstractImportTranslationsStep.java#L308 to `TMTextUnitVariant.Status status = null;` so that translations with `state="new"` wouldn't be imported. Would it break your workflow?

**Comment by aurambaj** [comment]
no, we don't import XLIFF. Where do we get files with "new" state though? When we export we should export only existing string so no "new" state, no?

**Comment by jeeyi** [comment]
When a drop is created, all strings that do not have translations get created with "new" state in the xliff. Then we expect translators to update the state from "new" to "translated" when they add translation, correct?

**Comment by aurambaj** [comment]
ha ok it is for the drop use case (I was thinking of xliff export/import). Ok then I see, if the drop is partially translated you want to skip the "new" strings which indeed looks like the intended behavior.
 
That said one think I can think of is that the state was not updated correctly (or not consistently) and that's why we imported the string regardless of the "new" state.

**Comment by jeeyi** [comment]
Fix: https://github.com/box/mojito/pull/391
